2025-09-05 15:32:26,602 - large_scale_cognitive_test_client_10 - INFO - Starting client 10 with config: {'client_id': 10, 'server_address': '0.0.0.0:8080', 'experiment_name': 'large_scale_cognitive_test', 'seed': 466, 'batch_size': 32, 'epochs': 2, 'learning_rate': 0.001, 'attack_enabled': False}
2025-09-05 15:32:35,082 - large_scale_cognitive_test_client_10 - INFO - Client 10 initialized with 5930 training samples
2025-09-05 15:32:35,084 - large_scale_cognitive_test_client_10 - INFO - Client 10 connecting to 0.0.0.0:8080
2025-09-05 15:32:35,085 - flwr - WARNING - DEPRECATED FEATURE: flwr.client.start_client() is deprecated.
	Instead, use the `flower-supernode` CLI command to start a SuperNode as shown below:

		$ flower-supernode --insecure --superlink='<IP>:<PORT>'

	To view all available options, run:

		$ flower-supernode --help

	Using `start_client()` is deprecated.

            This is a deprecated feature. It will be removed
            entirely in future versions of Flower.
        
2025-09-05 15:32:35,112 - flwr - DEBUG - Opened insecure gRPC connection (no certificates were passed)
2025-09-05 15:32:35,113 - flwr - DEBUG - ChannelConnectivity.IDLE
2025-09-05 15:32:35,114 - flwr - DEBUG - ChannelConnectivity.CONNECTING
2025-09-05 15:32:35,115 - flwr - DEBUG - gRPC channel closed
2025-09-05 15:32:35,115 - large_scale_cognitive_test_client_10 - ERROR - Client 10 failed: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:0.0.0.0:8080: Failed to connect to remote host: connect: Connection refused (61)"
	debug_error_string = "UNKNOWN:Error received from peer  {grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:0.0.0.0:8080: Failed to connect to remote host: connect: Connection refused (61)", grpc_status:14}"
>
2025-09-05 15:38:44,267 - large_scale_cognitive_test_client_10 - INFO - Starting client 10 with config: {'client_id': 10, 'server_address': '0.0.0.0:8080', 'experiment_name': 'large_scale_cognitive_test', 'seed': 466, 'batch_size': 32, 'epochs': 2, 'learning_rate': 0.001, 'attack_enabled': False}
2025-09-05 15:38:52,625 - large_scale_cognitive_test_client_10 - INFO - Client 10 initialized with 5930 training samples
2025-09-05 15:38:52,626 - large_scale_cognitive_test_client_10 - INFO - Client 10 connecting to 0.0.0.0:8080
2025-09-05 15:38:52,626 - flwr - WARNING - DEPRECATED FEATURE: flwr.client.start_client() is deprecated.
	Instead, use the `flower-supernode` CLI command to start a SuperNode as shown below:

		$ flower-supernode --insecure --superlink='<IP>:<PORT>'

	To view all available options, run:

		$ flower-supernode --help

	Using `start_client()` is deprecated.

            This is a deprecated feature. It will be removed
            entirely in future versions of Flower.
        
2025-09-05 15:38:52,652 - flwr - DEBUG - Opened insecure gRPC connection (no certificates were passed)
2025-09-05 15:38:52,655 - flwr - DEBUG - ChannelConnectivity.IDLE
2025-09-05 15:38:52,657 - flwr - DEBUG - ChannelConnectivity.TRANSIENT_FAILURE
2025-09-05 15:38:52,863 - flwr - DEBUG - gRPC channel closed
2025-09-05 15:38:52,864 - large_scale_cognitive_test_client_10 - ERROR - Client 10 failed: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:0.0.0.0:8080: Failed to connect to remote host: connect: Connection refused (61)"
	debug_error_string = "UNKNOWN:Error received from peer  {grpc_status:14, grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:0.0.0.0:8080: Failed to connect to remote host: connect: Connection refused (61)"}"
>
2025-09-05 15:41:29,352 - large_scale_cognitive_test_client_10 - INFO - Starting client 10 with config: {'client_id': 10, 'server_address': '20.169.237.127:8080', 'experiment_name': 'large_scale_cognitive_test', 'seed': 466, 'batch_size': 32, 'epochs': 2, 'learning_rate': 0.001, 'attack_enabled': False}
2025-09-05 15:41:37,688 - large_scale_cognitive_test_client_10 - INFO - Client 10 initialized with 5930 training samples
2025-09-05 15:41:37,688 - large_scale_cognitive_test_client_10 - INFO - Client 10 connecting to 20.169.237.127:8080
2025-09-05 15:41:37,688 - flwr - WARNING - DEPRECATED FEATURE: flwr.client.start_client() is deprecated.
	Instead, use the `flower-supernode` CLI command to start a SuperNode as shown below:

		$ flower-supernode --insecure --superlink='<IP>:<PORT>'

	To view all available options, run:

		$ flower-supernode --help

	Using `start_client()` is deprecated.

            This is a deprecated feature. It will be removed
            entirely in future versions of Flower.
        
2025-09-05 15:41:37,704 - flwr - DEBUG - Opened insecure gRPC connection (no certificates were passed)
2025-09-05 15:41:37,706 - flwr - DEBUG - ChannelConnectivity.IDLE
2025-09-05 15:41:37,706 - flwr - DEBUG - ChannelConnectivity.CONNECTING
2025-09-05 15:41:37,950 - flwr - DEBUG - ChannelConnectivity.TRANSIENT_FAILURE
2025-09-05 15:41:38,157 - flwr - DEBUG - gRPC channel closed
2025-09-05 15:41:38,158 - large_scale_cognitive_test_client_10 - ERROR - Client 10 failed: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:20.169.237.127:8080: Failed to connect to remote host: connect: Connection refused (61)"
	debug_error_string = "UNKNOWN:Error received from peer  {grpc_status:14, grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:20.169.237.127:8080: Failed to connect to remote host: connect: Connection refused (61)"}"
>
